target/linux/ramips/image/mt7620.mk

```diff
@@ -303,6 +303,15 @@ define Device/dlink_dwr-960
 endef
 TARGET_DEVICES += dlink_dwr-960
 
+define Device/domywifi_dw22d
+  SOC := mt7620a
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := DomyWifi
+  DEVICE_MODEL := DM202/DM203/DW22D
```

Comment:
DEVICE_MODEL should not provide more information that the actual (image) name.

If this is for three different devices that are sold under different names, please create three "devices" for them here. (Of course, creating DTSI files etc. will help to reduce duplicate code).

Reply:
These models are actually the same thing except the label. I just warry about these old devices will take up too much server storage resources. But if it's a unwritten rule I would like to change them.

Reply:
Well, one device should be one device. I never liked when the Wiki says "If you have device AAA please use the image for device XXX...."